lib/Auth/Basic.php

```diff
@@ -430,7 +530,7 @@ public function verifyCredentials($user, $password)
                 } elseif (strlen($hash) == 40) {
                     $this->password_encryption = 'sha1';
                 } else {
-                    $this->password_encryption = false;
+                    $this->password_encryption = null;
```

Comment:
ok. That was because I identified that password_encryption can e string or callable. false = bool, so in theory shouldn't be used here. But ... well, let's look at this some other time. I'll roll it back.